app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,4 @@
 # : true
-
 # : true
 
```

Comment:
- [ ]  Hi, excellent Job in writing the test. Please check on your test again because it has failed while checking.

![Capture](https://github.com/Victoria129/blog-app-rails/assets/63972670/6493d31e-0de7-4ef3-86d1-9fadd73b1823)